docs/whats-new.mdx

```diff
@@ -35,7 +42,7 @@ You can expect this page to update regularly (at least monthly). We'll include t
 - 2024-03-12 - [ngrok-python 1.2.0 Released](https://github.com/ngrok/ngrok-python/releases/tag/v1.2.0). [Changelog](https://github.com/ngrok/ngrok-python/blob/v1.2.0/CHANGELOG.md#120)
 - 2024-03-12 - [ngrok-javascript 1.2.0 Released](https://github.com/ngrok/ngrok-javascript/releases/tag/v1.2.0). [Changelog](https://github.com/ngrok/ngrok-javascript/blob/main/CHANGELOG.md#120)
 - 2024-03-11 - We added additional domains: `ngrok.pizza` and `ngrok.pro` for free and paid users respectively. Learn more in our [Introducing new domains for the ngrok global network](https://ngrok.com/blog-post/new-domains-global-network-launch) blog post.
-- 2024-03-07 - [ngrok-rust 0.14.0-pre12 Released](https://github.com/ngrok/ngrok-javascript/releases/tag/v1.1.1). [Changelog](https://github.com/ngrok/ngrok-rust/compare/ngrok-v0.14.0-pre.11...ngrok-v0.14.0-pre.12)
+- 2024-03-07 - [ngrok-rust 0.14.0-pre12 Released](https://github.com/ngrok/ngrok-rust/releases/tag/ngrok-v0.14.0-pre.12). [Changelog](https://github.com/ngrok/ngrok-rust/compare/ngrok-v0.14.0-pre.11...ngrok-v0.14.0-pre.12)
```

Comment:
@bobzilladev just confirming, this is fixing a bad link, right?

Reply:
Yeah, rust was pointing at javascript